internal/exec/workflow_utils.go

```diff
@@ -101,12 +101,37 @@ func ExecuteWorkflow(
 				logFunc(cliConfig, fmt.Sprintf("Stack: %s", finalStack))
 			}
 
-			if err := ExecuteShellCommand(cliConfig, "atmos", args, ".", []string{}, dryRun, ""); err != nil {
-				return err
-			}
+			err = ExecuteShellCommand(cliConfig, "atmos", args, ".", []string{}, dryRun, "")
 		} else {
 			return fmt.Errorf("invalid workflow step type '%s'. Supported types are 'atmos' and 'shell'", commandType)
 		}
+
+		if err != nil {
+			workflowFileName := filepath.Base(workflowPath)
+			workflowFileName = strings.TrimSuffix(workflowFileName, filepath.Ext(workflowFileName))
+
+			failedMsg := color.New(color.FgRed).Sprintf("\nStep '%s' failed!", step.Name)
+
+			u.LogDebug(cliConfig, fmt.Sprintf("\nCommand failed: %s", command))
+			u.LogDebug(cliConfig, fmt.Sprintf("Error: %v", err))
+
+			resumeWithCd := fmt.Sprintf("cd %s && atmos workflow %s -f %s --from-step %s",
```

Comment:
Looks like we are still printing the `cd %s` command. This doesn't make sense as the user should already be in that working directory.

Reply:
If anything we can print the working directory to the debug log